pkg/infra/metrics/frontendmetrics.go

```diff
@@ -0,0 +1,43 @@
+package metrics
+
+import "github.com/prometheus/client_golang/prometheus"
+
+// PostPostFrontendMetricsCommand sent by frontend to record frontend metrics
+type PostFrontendMetricsCommand struct {
+	Events []FrontendMetricEvent `json:"events"`
+}
+
+// FrontendMetricEvent a single metric measurement event
+type FrontendMetricEvent struct {
+	Name  string  `json:"name"`
+	Value float64 `json:"value"`
+}
```

Comment:
needs to be extended to support labels, but that will need some validation and list of predefined valid labels (to protect against attacks), so keeping it simple to begin with

Reply:
:+1: lets address that later :)